README.md

```diff
@@ -16,3 +16,19 @@ Current list of distributions:
 
 - [OpenTelemetry Collector (also known as "otelcol")](./distributions/otelcol)
 - [OpenTelemetry Collector Contrib (also known as "otelcol-contrib")](./distributions/otelcol-contrib)
+
+## Criteria for Distributions
+
```

Comment:
nit about wording/structure: I would start with

> Distributions supported by the Collector SIG should fulfill the following criteria

and remove this from the individual items to avoid repetiition.

It would also make it easier to discuss if the items are numbered (so that we can say "distro X does not fulfill requirement Y")

Reply:
Updated